jekyll/_cci2_ja/orb-author.md

```diff
@@ -21,19 +21,31 @@ version:
 
 Orb を初めて作成する方も、本番レベルで用意したい方も、[Orb 開発キット](#orb-development-kit)を使って Orb の開発を始めることをお勧めします。 または、Orb は[再利用可能な設定]({{site.baseurl}}/ja/reusing-config/)をパッケージにしたものなので、単体の `yaml` ファイルとして Orb を[手動で]({{site.baseurl}}/ja/orb-author-validate-publish)記述し、[CircleCI Orb 用の CLI]({{site.baseurl}}/ja/local-cli/#installation) を使用してパブリッシュすることも可能です。
 
-## Orb の作成、テスト、パブリッシュ
-{: #create-test-and-publish-an-orb }
 
-下記の手順に従って、Orb 開発キットを使って独自の Orb を作成、テスト、パブリッシュすることができます。
+## Orb 開発キット
+{: #orb-development-kit }
 
-Orb 開発キットは、相互に連携する複数のツールをセットにしたものです。キットを使うと CircleCI でのテストとデプロイが自動化されるため、Orb の開発プロセスが簡易化されます。 Orb 開発キットは、次の要素で構成されています。
+Orb 開発キットは、相互に連携する複数のツールをセットにしたものです。キットを使うと CircleCI でのテストとデプロイが自動化されるため、Orb の開発プロセスが簡易化されます。 The `orb init` command is the key to using the Orb Development Kit. This command initiates a new orb project based on a template, and that template uses the other tools in the kit to automatically test and deploy your orb.
+
+Orb 開発キットは、次の要素で構成されています。
 
 * [Orb テンプレート](https://github.com/CircleCI-Public/Orb-Template)
 * [CircleCI CLI](https://circleci-public.github.io/circleci-cli/)
     * [Orb Pack コマンド]({{site.baseurl}}/orb-concepts/#orb-packing)
     * [Orb Init コマンド](https://circleci-public.github.io/circleci-cli/circleci_orb_init.html)
 * [Orb ツールの Orb](https://circleci.com/developer/orbs/orb/circleci/orb-tools)
 
+The **orb template** is a repository with CircleCi's orb project template, which is automatically ingested and modified by the `orb init` command.
```

Comment:
```suggestion
The **orb template** is a repository with CircleCI's orb project template, which is automatically ingested and modified by the `orb init` command.
```

But I understand this is from the original text in English. No need to fix it here then!